src/project.rs

```diff
@@ -150,19 +150,23 @@ impl FuzzProject {
             cmd.arg("-Z").arg(flag);
         }
 
-        let mut rustflags: String = format!(
-            "--cfg fuzzing \
-             -Cpasses=sancov \
-             -Cllvm-args=-sanitizer-coverage-level=4 \
-             -Cllvm-args=-sanitizer-coverage-trace-compares \
-             -Cllvm-args=-sanitizer-coverage-inline-8bit-counters \
-             -Cllvm-args=-sanitizer-coverage-trace-geps \
-             -Cllvm-args=-sanitizer-coverage-prune-blocks=0 \
-             -Cllvm-args=-sanitizer-coverage-pc-table \
-             -Clink-dead-code \
-             -Zsanitizer={sanitizer}",
-            sanitizer = build.sanitizer,
-        );
+        let mut rustflags: String = "--cfg fuzzing \
+                                     -Cpasses=sancov \
+                                     -Cllvm-args=-sanitizer-coverage-level=4 \
+                                     -Cllvm-args=-sanitizer-coverage-trace-compares \
+                                     -Cllvm-args=-sanitizer-coverage-inline-8bit-counters \
+                                     -Cllvm-args=-sanitizer-coverage-trace-geps \
+                                     -Cllvm-args=-sanitizer-coverage-prune-blocks=0 \
+                                     -Cllvm-args=-sanitizer-coverage-pc-table \
+                                     -Clink-dead-code"
+            .to_owned();
+        if let Sanitizer::None = build.sanitizer {
+        } else {
```

Comment:
This is a bit funky -- let's change it to either

* a `match`, or
* add a `is_none` method to `Sanitizer` so we can do `if !build.sanitizer.is_none() { ... }` here.